src/plugin/places/kmlplacesimportui.js

```diff
@@ -156,14 +156,23 @@ plugin.places.KMLPlacesCtrl.prototype.onImportComplete_ = function(event) {
       i--;
     }
   }
-  // plugin.places.menu.saveKMLToPlaces_(nodes);
   var rootNode = plugin.places.PlacesManager.getInstance().getPlacesRoot();
   if (rootNode) {
     var cmds = [];
     for (var i = 0; i < nodes.length; i++) {
-      var cmd = new plugin.file.kml.cmd.KMLNodeAdd(nodes[i], rootNode);
-      cmd.title = 'Save ' + nodes[i].getLabel() + ' to Places';
-      cmds.push(cmd);
+      var feature = nodes[i].getFeature();
+      if (feature) {
+        if (nodes[i].annotation_) {
```

Comment:
You will want to call `getAnnotation()` like you did here: https://github.com/ngageoint/opensphere/pull/401/files so that this works in compiled mode.

Reply:
You shouldn't be interacting with the annotation outside of `KMLNode`, which is why I removed `getAnnotation`. Places should be imported exactly as they were exported, so there is no need to hide annotations manually here.

If you must hide it, the correct place to avoid conflicts with tree/animation visibility is the `show` property in the annotation options (`feature.get(os.annotation.OPTIONS_FIELD)`).